core/capabilities/ccip/launcher/deployment.go

```diff
@@ -32,17 +32,25 @@ type ccipDeployment struct {
 // Close shuts down all OCR instances in the deployment.
 func (c *ccipDeployment) Close() error {
 	var err error
+	// if this node doesn't have instances, it's already closed
+	if c == nil {
+		return nil
+	}
```

Comment:
Under which scenarios does this occur?

Reply:
I'm finalizing a test for this, but on [this line](https://github.com/smartcontractkit/chainlink/blob/39dcd54987cf0f48545c23e914cb8cf078e43522/core/capabilities/ccip/launcher/deployment.go#L110): 

It's possible for a node to be added to the don in a candidate configuration, then removed from the candidate configuration before ever having an active configuration.

Reply:
Can you add that as a comment on this code? That's good context to have when reading this.